tests/integration/testdata/validate/kong3x.yaml

```diff
@@ -0,0 +1,16 @@
+_format_version: "3.0"
```

Comment:
Can you also add consumers and some credentials? Feel free to do it here or in a different PR

Reply:
Added in this PR itself, in a diff file.